app/pages/ServiceDiscoveryForm/constants.ts

```diff
@@ -1,3 +1,12 @@
+export interface IServiceCategory {
+  algoliaCategoryName: string;
+  id: string;
+  name: string;
+  slug: string;
+  steps: string[];
```

Comment:
Done. It looks like the only thing we were using the STEPS const for was for cases in a switch statement in another file. I replaced the STEPS instances there with string literals, which does feel a bit weird. But it does makes sense to use enums rather than a string overall.